pegasus/sites.v3/code.org/views/interactive_map.haml

```diff
@@ -3,15 +3,27 @@
 
 - require 'state_abbr'
 
+- advocacy_variation ||= false
+- fill_color = advocacy_variation ? '#0094ca' : '#3E484F'
+- stroke_color = advocacy_variation ? 'white' : '#7A7A7A'
+
 #state-list.phone-feature
-  %h2 Choose a State
+  - if advocacy_variation
+    %h2 Choose your State
+  - else
+    %h2 Choose a State
+
   %form
     %select.dropdown#state-dropdown
       %option
       -STATE_ABBR_WITH_DC_HASH.each do |abbr, state|
         %option{value:abbr}= state
 
-%h2.tablet-feature Choose a State
+- if advocacy_variation
+  %h2.tablet-feature Choose your State
```

Comment:
can we consolidate these strings into variables since they're re-used? Or are you waiting for localization?

Reply:
For what it's worth, I think we'll switch to "Choose your State" for both advocacy.code.org and code.org before this goes live.